StudyHub/StudyHub/Program.cs

```diff
@@ -23,19 +22,18 @@
 using StudyHub.Validators.AssignmentTaskOptionValidators;
 using System.Text;
 using Hangfire;
-using StudyHub.Hangfire;
 using StudyHub.Hangfire.Extensions;
-using StudyHub.BLL.Configs;
+using StudyHub.Common.Configs;
 using Microsoft.AspNetCore.Mvc.ApplicationModels;
 using StudyHub.Utility;
 
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.Configure<JwtSettings>(builder.Configuration.GetSection("JwtSettings"));
-builder.Services.Configure<GoogleAuthConfig>(builder.Configuration.GetSection("GoogleAuth"));
-builder.Services.Configure<EmailSettings>(builder.Configuration.GetSection("EmailSettings"));
-builder.Services.Configure<HangfireConfig>(builder.Configuration.GetSection("HangfireConfig"));
-builder.Services.Configure<UserInvitationConfig>(builder.Configuration.GetSection("UserInvitationConfig"));
+builder.Services.AddConfig<JwtConfig>(builder.Configuration);
```

Comment:
I think we can avoid passing the same builder.Configuration for every AddConfig call